app/helpers/__init__.py

```diff
@@ -1,10 +1,22 @@
+from app.settings import CENSUS_CY_BASE_URL, CENSUS_EN_BASE_URL, CENSUS_NIR_BASE_URL
+
 from .address_lookup_api_helper import get_address_lookup_api_auth_token
 from .header_helpers import get_span_and_trace
-from .template_helpers import get_census_base_url
 from .url_safe_serializer import url_safe_serializer
 
+
+def get_base_url(schema_theme: str, language_code: str) -> str:
+    if language_code == "cy":
+        return CENSUS_CY_BASE_URL
+
+    if schema_theme == "census-nisra":
+        return CENSUS_NIR_BASE_URL
+
+    return CENSUS_EN_BASE_URL
```

Comment:
It won't. But at the same time, we have no requirements for how to select a base url for anything other than census (that I am aware of). When we do, this will need updating.

Reply:
Now removed this logic in favour of retrieving it from the SurveyConfig